Code/Include/HumanWorker/WaypointBus.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root
+ * of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Component/Component.h>
+#include <AzCore/Serialization/EditContext.h>
+
+namespace ROS2::HumanWorker
+{
+    struct WaypointConfiguration
```

Comment:
Can't we use a forward declaration scheme here so that the `WaypointConfiguration` does not have to be implemented in an interface header file?

Reply:
This is needed for an external gem (not yet released).

Reply:
Can we at least move it to a separate file, so it is not implemented in a bus header?